usr/src/contrib/libfdt/libfdt_env.h

```diff
@@ -7,17 +7,34 @@
  * Copyright 2012 Kim Phillips, Freescale Semiconductor.
  */
 
+#if defined(_STANDALONE)
+#include <stand.h>
+#include <stdbool.h>
```

Comment:
These end up being boot/ includes, not those from the system right?

Reply:
That's right, _STANDALONE refers to the `usr/src/boot` standalone.